package.json

```
@@ -26,7 +26,7 @@
   "dependencies": {
     "chalk": "^2.3.0",
     "lodash": "^4.17.11",
-    "node-sass": "^4.8.3",
+    "node-sass": "^6.0.0",
```

There's no reason to drop support for old versions.
```suggestion
    "node-sass": ">=4.8.3 && <7",
```